src/widget/wbeatspinbox.h

```diff
@@ -31,6 +31,19 @@ class WBeatSpinBox : public QDoubleSpinBox, public WBaseWidget {
 
     ControlProxy m_valueControl;
 
+    // for L/R button BeatJump/LoopMove
+    ControlProxy beatJumpForward;
+    ControlProxy beatJumpBackward;
+    ControlProxy beatJumpSize;
+    ControlProxy beatJumpOneForward;
+    ControlProxy beatJumpOneBackward;
+    ControlProxy beatJumpEightForward;
+    ControlProxy beatJumpEightBackward;
+    ControlProxy beatLoopEnabled;
+    ControlProxy beatLoopSize;
+    QSize sizeSpinBox;
+    QSize sizeLoopBox;
```

Comment:
For class members we use `m_` prefix.
I suggest you take a look at https://github.com/mixxxdj/mixxx/wiki/Coding-Guidelines